app/src/main/kotlin/com/quittle/a11yally/analyzer/linearnavigation/LinearNavigationState.kt

```diff
@@ -46,7 +46,7 @@ data class LinearNavigationState(
     fun haveEntriesDiverged(): Boolean {
         val curText = entries.map(LinearNavigationEntry::text)
         val prevText = prevEntries.map(LinearNavigationEntry::text)
-        val intersection = curText.intersect(prevText)
+        val intersection = curText.intersect(prevText.toSet())
```

Comment:
Why did this need to be converted to a set? `.intersect` should take any number of types.

Reply:
I see now, Android Studio recommends it. It's a bit silly since it's pointless work but I don't mind it for now.

Reply:
Yes, Android Studio hinted that "The argument can be converted to 'Set' to improve performance" so I didn't hesitate to change it a little bit.